redash/query_runner/clickhouse.py

```diff
@@ -66,7 +71,7 @@ def _send_query(self, data, stream=False):
             self.configuration['url'],
             data=data.encode("utf-8"),
             stream=stream,
-            timeout=5,
+            timeout=self.configuration['timeout'],
```

Comment:
This should be `self.configuration.get('timeout', 30)` as otherwise it will break if no value is set.

Reply:
Oops. Fixed